incubator/jaeger/templates/query-deploy.yaml

```diff
@@ -36,89 +36,92 @@ spec:
+              value: {{ .Values.storage.cassandra.user }}
+            - name: CASSANDRA_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ if .Values.storage.cassandra.existingSecret }}{{ .Values.storage.cassandra.existingSecret }}{{- else }}{{ include "jaeger.fullname" . }}-cassandra{{- end }}
+                  key: password
+            {{- end }}
+            {{- if eq .Values.storage.type "elasticsearch" }}
+            - name: ES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ if .Values.storage.elasticsearch.existingSecret }}{{ .Values.storage.elasticsearch.existingSecret }}{{- else }}{{ include "jaeger.fullname" . }}-elasticsearch{{- end }}
+                  key: password
+            - name: ES_SERVER_URLS
+              value: {{ template "elasticsearch.client.url" . }}
+            - name: ES_USERNAME
+              value: {{ .Values.storage.elasticsearch.user }}
+            {{- end }}
+            - name: QUERY_BASE_PATH
+              value: {{ .Values.query.basePath | quote }}
+          ports:
+            - name: query
+              containerPort: 16686
+              protocol: TCP
+            - name: admin
+              containerPort: 16687
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: admin
+          readinessProbe:
+            httpGet:
+              path: /
+              port: admin
+          resources:
+            {{ toYaml .Values.query.resources | indent 12 }}
 {{- if .Values.query.agentSidecar.enabled }}
-      - name: {{ template "jaeger.agent.name" . }}-sidecar
-        image: {{ .Values.agent.image }}:{{ .Values.tag }}
-        imagePullPolicy: {{ .Values.agent.pullPolicy }}
-        env:
-        - name: REPORTER_GRPC_HOST_PORT
-          value: {{ include "jaeger.collector.name" . }}:{{ .Values.collector.service.grpcPort }}
-        ports:
-        - name: admin
-          containerPort: 14271
-          protocol: TCP
-        livenessProbe:
-          httpGet:
-            path: /
-            port: admin
-        readinessProbe:
-          httpGet:
-            path: /
-            port: admin
+        - name: {{ template "jaeger.agent.name" . }}-sidecar
+          securityContext:
+            {{- toYaml .Values.agent.securityContext | nindent 12 }}
+          image: {{ .Values.agent.image }}:{{ .Values.tag }}
+          imagePullPolicy: {{ .Values.agent.pullPolicy }}
+          env:
+          - name: REPORTER_GRPC_HOST_PORT
+            value: {{ include "jaeger.collector.name" . }}:{{ .Values.collector.service.grpcPort }}
+          ports:
+          - name: admin
+            containerPort: 14271
+            protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: admin
+          readinessProbe:
+            httpGet:
+              path: /
+              port: admin
```

Comment:
This whole indentation is unrelated change. Please, revert it.

Reply:
My intention is that charts be somewhat standardized. The helm create command uses an indentation for lists. I actually preferred no indentation and made a PR helm/helm#5725 ... However after getting feedback from at least 2 helm maintainers and some of my colleagues, the general consensus is that indented lists are more readable.

Reply:
But it is indeed unrelated to security context